frontend/packages/dev-console/integration-tests/utilities/appFunctions.ts

```diff
@@ -0,0 +1,71 @@
+import {click} from './elementInteractions';
+import {$, browser, ExpectedConditions as EC} from 'protractor';
+const waitForElement = 5000;
+
+export enum NavigationMenu {
```

Comment:
No Pipelines?

Reply:
Pipelines are not displaying in developer view. If it displays will include them.. 
This code will be enhanced on adding additional features (left side navigation menu)